pkg/http/retry.go

```diff
@@ -0,0 +1,66 @@
+package http
+
+import (
+	"bytes"
+	"io"
+	"math"
+	"net/http"
+	"time"
+)
+
+type RetryableTransport struct {
+	Transport  http.RoundTripper
+	RetryCount int
+}
+
+func (t *RetryableTransport) RoundTrip(req *http.Request) (*http.Response, error) {
```

Comment:
Please add test cases for this too

Reply:
Added test cases for the function

Reply:
`Timeout` is the part of  new HTTP client from Go package. Let's just remove the const and use the default from the http package (90 secs). But still this 10 secs should be set on config since 90 is too much for the timeout.